### PR TITLE
Update Ukrainian translation

### PR DESCRIPTION
**Pull request by elsieholmes**
This PR fixes numerous mistakes in different translations, adds missing Ukrainian translations, improves the general style, and replaces regional slang with adequate words.

### CODE REVIEW

File: app/javascript/mastodon/locales/en.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
   "emoji_button.food": "Food & Drink",
   "emoji_button.label": "Insert emoji",
   "emoji_button.nature": "Nature",
-  "emoji_button.not_found": "No emojos!! (╯°□°）╯︵ ┻━┻",
+  "emoji_button.not_found": "No emojis!! (╯°□°）╯︵ ┻━┻",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/en.json]: https://github.com/tootsuite/mastodon/pull/5002#discussion_r139579169

**Comment by elsieholmes** [comment]
@ykzts Fixed. Enjoy.

**Comment by ykzts** [comment]
cc @stamak @sterdev

**Comment by elsieholmes** [comment]
Oh really? Editing despite disallowing edits from maintainers? 
Maybe you'll also try to attack bloody cryptographic commit signatures? 😠
> I'd wait for someone official to approve as well.

It's my second language. Just for your information, I've successfully passed a goddamned state exam in Ukraine itself. If that's not enough, I have no idea what. Don't want to? No need to. 
Fare thee well. That was the last straw!

**Comment by ykzts** [comment]
Modification of English typo has been removed from this Pull Request. For that reason I edited the title for management. Is this wrong?

**Comment by elsieholmes** [comment]
I don't care. Самі їжте своє лайно, я не братиму в цьому участі.

**Comment by larouxn** [comment]
Wait, what? I'm just new to this repo and thus don't feel like I have the authority to approve PRs yet.

As for the language thing, I'm sure your Ukranian is fine. It wasn't a comment on your content but rather that I cannot speak to whether it's correct or not as I don't know any Ukranian myself.

My apologies for any misunderstandings.

**Comment by elsieholmes** [comment]
I hereby officially **FORBID** you to use these changes! 
DO NOT INCLUDE OR ALTER THEM!

**Pull request closed without merging**